### ISSUE TITLE
Add missing tests for UnusedReturnValueAnalyzer

### ISSUE THREAD

**Issue by adamralph** [FakeItEasy/FakeItEasy#765]
Not all the [listed methods](https://github.com/FakeItEasy/FakeItEasy/blob/508987ae8aa5599d29ea09668fab744a76d2d4e5/src/FakeItEasy.Analyzer/UnusedReturnValueAnalyzer.cs#L72-L82) are covered.

**Comment by blairconrad** [comment]
It looks like we're missing

```c#
FakeItEasy.Fake`1.CallsTo`1
FakeItEasy.Fake`1.AnyCall
FakeItEasy.Configuration.IAnyCallConfigurationWithNoReturnTypeSpecified.WithReturnType`1
FakeItEasy.Configuration.IArgumentValidationConfiguration`1.WhenArgumentsMatch
```

**Comment by medmondson** [comment]
This is something I can pick up

**Comment by thomaslevesque** [comment]
> This is something I can pick up

Thanks @medmondson! Let us know if you need any help.

**Comment by blairconrad** [comment]
Thanks, @medmondson!

**Comment by blairconrad** [comment]
This change has been released as part of [FakeItEasy 3.1.0](https://github.com/FakeItEasy/FakeItEasy/releases/tag/3.1.0).

@medmondson, look for your name in the release notes! :trophy:

**Comment by medmondson** [comment]
Thanks very much, glad to be of help
